mqtt/mqtt-bridge/Cargo.toml

```diff
@@ -16,7 +16,8 @@ futures-util = "0.3"
 humantime = "2.0"
 humantime-serde = "1.0"
 lazy_static = "1.4"
-mockall = "0.8"
+mockall = "0.9"
```

Comment:
Can this not be a dev dependency and just have mockall_double as the normal dependency (as it is required for doubling)?

Reply:
Great question! I tried that, but since we annotate production code with `#[automock]` macro I cannot move it as dev-dependency.